lib/CommandImplementations.ts

```diff
@@ -150,6 +150,22 @@ const states = [
     "West Virginia",
 ];
 
+export async function replyWithMention(msg: Message, reply: string): Promise<void> {
+    if (msg.mentions.users.size > 0 || msg.mentions.repliedUser)   {
+        let usersMentioned = "";
+        msg.mentions.users.forEach((value, key) => {
+            usersMentioned = '<@'+value+'> ' + usersMentioned;
+        })
+        if (msg.mentions.repliedUser)   {
+        usersMentioned = '<@'+msg.mentions.repliedUser+'> ' + usersMentioned;
+        }
```

Comment:
This block is un-needed, a reply will show up in `msg.mentions`, this will result in the user being tagged twice